graylog2-rest-client/src/main/java/org/graylog2/restclient/models/Stream.java

```diff
@@ -204,6 +206,7 @@ public Boolean getDisabled() {
         return (disabled != null && disabled);
     }
 
+    @JsonIgnore
```

Comment:
I think adding `@JsonIgnore` to private methods isn't necessary, especially since this isn't just a normal getter and won't be serialized by Jackson anyway.